github/src/main/kotlin/gropius/sync/github/repository/IssueInfoRepository.kt

```diff
@@ -0,0 +1,30 @@
+package gropius.sync.github.repository
+
+import gropius.sync.github.model.IssueInfo
+import kotlinx.coroutines.flow.Flow
+import org.bson.types.ObjectId
+import org.springframework.data.mongodb.repository.ReactiveMongoRepository
+import org.springframework.stereotype.Repository
+
+/**
+ * Mapping of a single issue from neo4j to github
+ */
+@Repository
+interface IssueInfoRepository : ReactiveMongoRepository<IssueInfo, ObjectId> {
+    /**
+     * Find all issues that have outstanding changes
+     */
+    suspend fun findByDirtyIsTrue(): Flow<IssueInfo>
```

Comment:
missing documentation for params, return

Reply:
4428e298d423fb8732889ff48146dcffe4724df9